metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -380,11 +388,16 @@
                                      (where `(and (= :c.concept-id :t.concept-id)
                                                   (= :c.revision-id :t.revision-id)))))
 
-              result (doall (map (partial db-result->concept-map concept-type conn provider-id)
-                                 (su/query conn stmt)))
+              result-oracle (if (not (= "efs-only" efs-config/efs-toggle))
+                              (doall (map (partial db-result->concept-map concept-type conn provider-id)
+                                          (su/query conn stmt)))
+                              nil)
+              result-efs (if (not (= "efs-off" efs-config/efs-toggle))
```

Comment:
You should use (when instead of (if in this case.  When returns nil if not true.